observability-lib/core-node/component.go

```diff
@@ -1058,7 +1073,7 @@ func nodesRPC(p Props) []cog.Builder[dashboard.Panel] {
 		common.BigValueTextModeValueAndName,
 		common.VizOrientationHorizontal,
 		utils.PrometheusQuery{
-			Query:  `sum(multi_node_states{` + p.PlatformOpts.LabelQuery + `chainId=~"$evmChainID", state="Alive"}) by (` + p.PlatformOpts.LegendString + `, chainId)`,
+			Query:  `sum(multi_node_states{` + p.PlatformOpts.LabelQuery + `state="Alive"}) by (` + p.PlatformOpts.LegendString + `, chainId)`,
```

Comment:
How about capturing these fields (or the whole PlatformOpts) in vars to make the queries a little more readable?
```suggestion
			Query:  `sum(multi_node_states{` + labelQuery + `state="Alive"}) by (` + legendString + `, chainId)`,
```
```suggestion
			Query:  `sum(multi_node_states{` + o.LabelQuery + `state="Alive"}) by (` + o.LegendString + `, chainId)`,
```

Reply:
its quite a big refactor, I could do it but in another PR as its not the scope of this one